test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -6,6 +6,7 @@ def setup
     @cftgateway = WorldpayGateway.new(fixtures(:world_pay_gateway_cft))
 
     @amount = 100
+    @year = (Time.now.year + 2).to_s[-2..-1].to_i
```

Comment:
Nice. It may be controversial, but I'm a fan of negative indices in Ruby. Very straightforwards and readable.